administrator/templates/atum/html/layouts/chromes/well.php

```diff
@@ -71,7 +70,7 @@
 					<?php endif; ?>
 
 					<?php if ($module->showtitle) : ?>
-						<h2 class="<?php echo $headerClass; ?>"><?php echo $headerIcon . htmlspecialchars($module->title); ?></h2>
+						<h2 <?php echo $headerClass; ?>><?php echo $headerIcon . htmlspecialchars($module->title); ?></h2>
```

Comment:
```suggestion
						<h2<?php echo $headerClass; ?>><?php echo $headerIcon . htmlspecialchars($module->title); ?></h2>
```

Reply:
Removing the space from here and adding them to the $headerClass will prevent you getting `<h2 >`